raco/representation.py

```diff
@@ -0,0 +1,17 @@
+__author__ = 'brandon'
+
+
+class RepresentationProperties(object):
+    def __init__(self, hash_partitioned=set(), sorted=None, grouped=None):
+        """
+        @param hash_partitioned: None or list of AttributeRefs in hash key
```

Comment:
Isn't this inaccurate given that `hash_partitioned` is an empty set by default and is also a set, not a list?